static/css/style.css

```diff
@@ -116,6 +123,25 @@ a:hover {
 
 /* Entry Review Page */
 
+.entry-container {
+    margin-top: 2rem;
+}
+
+.delete-entry {
+    text-transform: uppercase;
+}
+
+.entry .delete-btn {
+    font-size: 1.5rem;
+    padding-bottom: 0.2rem;
+    transition: all 0.3s;
```

Comment:
If this is something you would like across the board, you could issue a global declaration at the beginning of your spreadsheet, such as:
```css
* {
  transition-duration: 0.3s;
}
```